graylog2-server/src/main/resources/org/graylog2/plugin/journal/raw_message.proto

```diff
@@ -1,8 +1,10 @@
+syntax = "proto2";
+
 package org.graylog2.plugin.journal;
 
-// to regenerate:
-// install protobuf compiler (this was generated with 2.5.0)
```

Comment:
Let's please generate the `JournalMessages` with protobuf compiler version 2.5.0 to avoid the big diff.